README.md

```diff
@@ -18,12 +18,13 @@ For Rails 3, use gem version 3.0 or see the [3.0-stable branch](https://github.c
 
 Audited supports and is [tested against](https://github.com/collectiveidea/audited/actions/workflows/ci.yml) the following Ruby versions:
 
-* 2.3
+* 2.3 (only tested on Sqlite due to testing issues with other DBs)
```

Comment:
Makes sense.

Ruby 2.3 was [end-of-lifed over 3 years ago](https://www.ruby-lang.org/en/news/2019/03/31/support-of-ruby-2-3-has-ended/). The [`pg` gem](https://github.com/ged/ruby-pg) dropped Ruby 2.3 and 2.4 support in the [1.3.0 release](https://github.com/ged/ruby-pg/blob/master/History.rdoc#v130-2022-01-20-michael-granger-gedfaeriemudorg-).

Tracking down a fixing a segfault coming from a dependent gem for outdated and unsupported versions doesn't seem worth the effort.